packages/compass-sidebar/test/renderer/components/sidebar-instance.spec.jsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
+import { expect } from 'chai';
 import { mount } from 'enzyme';
 import SidebarInstance from '../../../src/components/sidebar-instance';
 import styles from '../../../src/components/sidebar-instance/sidebar-instance.module.less';
 
-describe('SidebarInstance [Component]', () => {
+describe.skip('SidebarInstance [Component]', function () {
```

Comment:
Is this (and a few skips above) intended?

Reply:
Yes as explained in the PR description these renderer tests were all skipped for ci and already broken/rotten.

Reply:
(Btw I did double check by running those tests on main - crashed before it ever got to the tests then ran zero tests.)

Reply:
Ahhh, my bad, didn't connect the two for some reason, thanks for clarifying!